### PR TITLE
added one Dr. Dre quote

### PR DESCRIPTION
**Pull request by vicowiti**
# Pull Request

## Related Issue:

Closes #XXXXX

### CODE REVIEW

File: src/config/QuoteList.ts [hunk 1]
```diff
@@ -565,4 +565,8 @@ export const QuoteList: Quote[] = [
     text: "Education is not the filling of a pail, but the lighting of a fire.",
     author: "William Butler Yeats",
   },
+  {
+    text: "And even when I was close to defeat, I rose to my feet."
```
Comment (nhcarrigan) [inline src/config/QuoteList.ts:569]:
```suggestion
    text: "And even when I was close to defeat, I rose to my feet.",
```
Missing comma here :)

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

File: src/config/QuoteList.ts (hunk 1 above)
Comment (vicowiti) [reply src/config/QuoteList.ts]: Added the missing comma, thanks :)

Comment (vicowiti) [reply src/config/QuoteList.ts]: In the latest commit

**Review by nhcarrigan — APPROVED** [review]
Looks good to me!

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Pull request merged**